cmake/utils.cmake

```
@@ -323,29 +323,34 @@ function(momo_strip_target target)
     return()
   endif()
 
-  if(NOT MSVC)
-    # TODO: detect LLVM IR bitcode and abort
+  if(MSVC OR MOMO_ENABLE_SANITIZER)
```

probably also disable it for android and ios for now 😂
```suggestion
  if(MSVC OR MOMO_ENABLE_SANITIZER OR CMAKE_SYSTEM_NAME STREQUAL "iOS" OR CMAKE_SYSTEM_NAME STREQUAL "Android")
```